language-support/hs/bindings/src/DA/Ledger/Services/CommandService.hs

```diff
@@ -59,7 +53,7 @@ submitAndWaitForTransaction commands =
             Right response ->
                 either (fail . show) (return . Right) $ raiseResponse response
             Left details ->
-                return $ Left $ show $ unStatusDetails details
+                return $ Left details
```

Comment:
We have quite a few functions returning `LedgerService (Either _ _)`. Would it make sense to move the error handling capabilities into the `LedgerService` monad?

Reply:
I don't think so.